bot/database/response_keys.py

```diff
@@ -0,0 +1,80 @@
+from enum import Enum
+
+
```

Comment:
tu taka luzna sugestia, sprobuj sobie wyrownac znak `=` miedzy linijakmi, ciekawa konwencja ktora niedawno poznalem, imo poprawia czytelnosc ale you do you, tutaj pasuje jak ulal

Reply:
czyli np.

```python3
a            = "tekst"
cos_dlugiego = "a"
```